templates/collections.html

```diff
@@ -29,7 +29,11 @@ <h1>Collections</h1>
 
 				<p> <a href = "{{url_for('deleteCollection',collection_id = collection.id)}}">Delete Collection</a> </p>
 
-				<img src="https://ak8.picdn.net/shutterstock/videos/30307858/thumb/1.jpg" alt="Yoga">
+				<!-- The below example uses Python string interpolation -->
+				<img src="{{ url_for('static', filename='images/{image_path}.jpg'.format(image_path=collection.name.lower()))}}" alt="{{collection.name}}">
```

Comment:
instead of hardcoding the image path, I am dynamically building the url to the [image assets using the `url_for` method with the `static` argument](http://stackabuse.com/serving-static-files-with-flask/) and [interpolating](https://stackoverflow.com/questions/4450592/is-there-a-python-equivalent-to-rubys-string-interpolation) and [lowercasing](https://stackoverflow.com/questions/6797984/how-to-convert-string-to-lowercase-in-python) the `collection.name` value (for consistency) which then matches up to the assets in the `static/images` directory.

For example, let's say the first collection in the for loop is the `run` collection:
1. `collection.name` will be `Run`
2. in the `url_for` method, we're interpolating the value of `collection.name.lower()` which is `run` (note the lower case)
3. `url_for('static'...` will automatically look for assets in the `static` dir
4. I moved the `images` dir inside the `static` dir, and am pointing to in inside the `filename` argument
5. the resulting image path is now `/static/images/run.jpg`
6. This happens dynamically for each value in `collections`